lib/base_wizard.py

```diff
@@ -137,7 +137,7 @@ def import_addresses_or_keys(self):
         title = _("Import Bitcoin Addresses")
         message = _("Enter a list of Bitcoin Cash addresses (this will create a watching-only wallet), or a list of private keys.")
         if bitcoin.is_bip38_available():
-            message += " " + _("BIP38 encrpted keys are supported.")
+            message += " " + _("BIP38 encyrpted keys are supported.")
```

Comment:
Oops. That was my typo. Thanks for this.

Reply:
Although you continued to typo it -- `encrypted` -- you have `encyrpted` :) Typos are contagious sometimes. :)

Reply:
Haha, that's hilarious. I'm so done with typing.

Reply:
Yeah there is a known phenomenon that typos are contagious. Thanks for catching this. I do this more than I like to admit in my user-facing messages.